tests/test_npm_meta_s3.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+import boto3
+from moto import mock_s3
+
+from mrrc.config import mrrc_config, AWS_ENDPOINT
+from mrrc.pkgs.npm import store_package_metadata_to_S3, read_package_metadata_from_content
+from mrrc.storage.s3client import S3Client
+from tests.base import BaseMRRCTest
+
+MY_BUCKET = "npm_bucket"
+
+
+@mock_s3
+class NPMMetadataTest(BaseMRRCTest):
+    def setUp(self):
+        super().setUp()
+        self.mock_s3 = self.__prepare_s3()
+        self.mock_s3.create_bucket(Bucket=MY_BUCKET)
+        self.s3_client = S3Client()
+
+    def tearDown(self):
+        s3 = self.__prepare_s3()
+        bucket = s3.Bucket(MY_BUCKET)
+        bucket.objects.all().delete()
+        super().tearDown()
+
+    def __prepare_s3(self):
+        conf = mrrc_config()
+        aws_configs = conf.get_aws_configs()
+        return boto3.resource(
+            "s3",
+            region_name=conf.get_aws_region(),
+            aws_access_key_id=conf.get_aws_key_id(),
+            aws_secret_access_key=conf.get_aws_key(),
+            endpoint_url=aws_configs[AWS_ENDPOINT] if AWS_ENDPOINT in aws_configs else None
+        )
+
+    def test_store_package_metadata_to_S3_for_old_version(self):
+        bucket = self.mock_s3.Bucket(MY_BUCKET)
+        original_old_version_package_json = \
```

Comment:
Suggest to use python string literal here: """ content """. This can avoid character escape (like \\" here) and support line break too, which will make your string content more readable.

Reply:
@ligangty thank you, copy that.